src/packages/media/media/components/input-image-cropper/image-cropper-focus-setter.element.ts

```diff
@@ -79,73 +93,145 @@ export class UmbImageCropperFocusSetterElement extends LitElement {
+
+		handle?.focus();
 		event.preventDefault();
-		if (this.hideFocalPoint) return;
+		event.stopPropagation();
 
-		if (!this.focalPointElement || !this.imageElement) return;
+		this.isDraggingGridHandle = true;
 
-		const image = this.imageElement.getBoundingClientRect();
+		drag(grid, {
+			onMove: (x, y) => {
+				// check if coordinates are not NaN (can happen when dragging outside of the grid)
+				if (isNaN(x) || isNaN(y)) return;
 
-		const x = clamp(event.clientX - image.left, 0, image.width);
-		const y = clamp(event.clientY - image.top, 0, image.height);
+				this.coords.x = x;
+				this.coords.y = y;
 
-		const left = clamp(x / image.width, 0, 1);
-		const top = clamp(y / image.height, 0, 1);
+				this.#setFocalPoint(x, y, width, height);
+			},
+			onStop: () => (this.isDraggingGridHandle = false),
+			initialEvent: event,
+		});
+	}
 
-		this.focalPointElement.style.left = `calc(${left * 100}% - ${this.#DOT_RADIUS}px)`;
-		this.focalPointElement.style.top = `calc(${top * 100}% - ${this.#DOT_RADIUS}px)`;
+	#handleGridKeyDown(event: KeyboardEvent) {
+		if (this.disabled || this.hideFocalPoint) return;
+		
+		const increment = event.shiftKey ? 1 : 10;
 
-		this.dispatchEvent(
-			new CustomEvent('change', {
-				detail: { left, top },
-				bubbles: false,
-				composed: false,
-			}),
-		);
+		const grid = this.wrapperElement;
+		if (!grid) return;
+
+		const { width, height } = grid.getBoundingClientRect();
+
+		if (event.key === 'ArrowLeft') {
+			event.preventDefault();
+			this.coords.x = clamp(this.coords.x - increment, 0, width);
+			this.#setFocalPoint(this.coords.x, this.coords.y, width, height);
+		}
+
+		if (event.key === 'ArrowRight') {
+			event.preventDefault();
+			this.coords.x = clamp(this.coords.x + increment, 0, width);
+			this.#setFocalPoint(this.coords.x, this.coords.y, width, height);
+		}
+
+		if (event.key === 'ArrowUp') {
+			event.preventDefault();
+			this.coords.y = clamp(this.coords.y - increment, 0, height);
+			this.#setFocalPoint(this.coords.x, this.coords.y, width, height);
+		}
+
+		if (event.key === 'ArrowDown') {
+			event.preventDefault();
+			this.coords.y = clamp(this.coords.y + increment, 0, height);
+			this.#setFocalPoint(this.coords.x, this.coords.y, width, height);
+		}
 	}
 
 	override render() {
 		if (!this.src) return nothing;
 		return html`
-			<div id="wrapper">
-				<img id="image" @click=${this.#onSetFocalPoint} @keydown=${() => nothing} src=${this.src} alt="" />
-				<div id="focal-point" class=${this.hideFocalPoint ? 'hidden' : ''}></div>
+			<div id="wrapper"
+				@mousedown=${this.#handleGridDrag}
+        			@touchstart=${this.#handleGridDrag}>
+				<img id="image" @keydown=${() => nothing} src=${this.src} alt="" />
+				<span id="focal-point"
+					class=${classMap({
+						'focal-point--dragging': this.isDraggingGridHandle,
+						'hidden': this.hideFocalPoint
+					})}
+					tabindex=${ifDefined(this.disabled ? undefined : '0')}
+					aria-label="Focal Point"
```

Comment:
We need a localization key for this label, perhaps in the "visuallyHiddenTexts" category.

Reply:
Does it need to extends `UmbLitElement` instead of `LitElement` to access localize?

```
aria-label="${this.localize.term('visuallyHiddenTexts_focalPoint')}"
```

If just "Focal Point" we can eventually use `general_focalPoint`?

Reply:
Yes and yes.  You need UmbLitElement, which it should have been in the first place. Actually made the same change to UmbLitElement over in #2145.

Reply:
Okay, makes sense.. I will have a look when https://github.com/umbraco/Umbraco.CMS.Backoffice/pull/2145 has been merged :)

Reply:
I have updated this and a few other labels.